akka-stream/src/main/scala/akka/stream/impl/ActorPublisher.scala

```diff
@@ -16,8 +16,9 @@ import org.reactivestreams.Subscription
  * INTERNAL API
  */
 private[akka] object ActorPublisher {
-  class NormalShutdownException extends IllegalStateException("Cannot subscribe to shut-down Publisher") with NoStackTrace
-  val NormalShutdownReason: Option[Throwable] = Some(new NormalShutdownException)
+  val NormalShutdownReasonMessage = "Cannot subscribe to shut-down Publisher"
+  class NormalShutdownException extends IllegalStateException(NormalShutdownReasonMessage) with NoStackTrace
+  val NormalShutdownReason: Throwable = new NormalShutdownException
```

Comment:
The previous value was optimized for not allocating a `Some` during each shutdown, perhaps we can retain this as `val SomeNormalShutdownReason`?